sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/RequestValidator.java

```diff
@@ -40,62 +40,59 @@ private RequestValidator() {}
    */
   @SuppressWarnings("unchecked")
   public static void validateUpdateContent(Request contentRequestDto) {
-    if (((List<Map<String, Object>>) (contentRequestDto.getRequest().get(JsonKey.CONTENTS)))
-        .isEmpty()) {
-      throw new ProjectCommonException(
-          ResponseCode.contentIdRequired.getErrorCode(),
-          ResponseCode.contentIdRequiredError.getErrorMessage(),
-          ERROR_CODE);
-    } else {
+
+    List<Map<String, Object>> contents =  (List<Map<String, Object>>) contentRequestDto.getRequest().get(JsonKey.CONTENTS);
+    if (!CollectionUtils.isEmpty(contents)) {
       List<Map<String, Object>> list =
-          (List<Map<String, Object>>) (contentRequestDto.getRequest().get(JsonKey.CONTENTS));
+              (List<Map<String, Object>>) (contentRequestDto.getRequest().get(JsonKey.CONTENTS));
```

Comment:
if list is null then what will happen

Reply:
CollectionUtils .isNotEmpty will handle null